### PR TITLE
[OSDOCS-4767] Add Kuryr deprecation module to some assemblies

### PR DESCRIPTION
**Pull request by maxwelldb**
Version(s): 4.12

Issue: https://issues.redhat.com/browse/OSDOCS-4767

Links to docs previews:
- https://54414--docspreview.netlify.app/openshift-enterprise/latest/networking/load-balancing-openstack.html
- https://54414--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_openstack/installing-openstack-user-kuryr.html
- https://54414--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_openstack/installing-openstack-user-kuryr.html

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by dulek — CHANGES_REQUESTED** [review]

File: networking/load-balancing-openstack.adoc [hunk 1]
```diff
@@ -6,6 +6,9 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
+:FeatureName: Kuryr
+include::modules/deprecated-feature.adoc[leveloffset=+1]
+
```
[the remark this answers is not in the document]
Comment (maxwelldb) [reply networking/load-balancing-openstack.adoc:11]: @dulek So, it seems like I have some play with the banner as long as it's thoughtful. 

What do you mean by "that section" specifically, and is it different from "the warning"? 

Like, some part of the deprecated-feature banner is at the top of the page, and the rest is elsewhere, or the banner stays as-is and there are smaller notes placed throughout the page?

[the remark this answers is not in the document]
Comment (dulek) [reply networking/load-balancing-openstack.adoc:11]: "That section" is `nw-osp-configuring-external-load-balancer.adoc`. This one is independent from Kuryr, so shouldn't be subject of the deprecation. Putting the banner at the top suggests it applies to all the content on this page, which won't be true.

**Review by dulek — CHANGES_REQUESTED** [review]

File: networking/load-balancing-openstack.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (maxwelldb) [reply networking/load-balancing-openstack.adoc:11]: Ah, got it. 

I now have permission to change how we do deprecation content to make this easier. Will need to make some changes in another PR and the incorporate them here. NBD, but will take an hour or so.

[the remark this answers is not in the document]
Comment (maxwelldb) [reply networking/load-balancing-openstack.adoc:11]: @dulek Getting this merged will allow more freedom here: https://github.com/openshift/openshift-docs/pull/54434

Shouldn't take long.

**Comment by maxwelldb** [comment]
@dulek This shows the deprecation snippet inserted into some of the Kuryr sections: https://54414--docspreview.netlify.app/openshift-enterprise/latest/networking/load-balancing-openstack.html

Thoughts?

**Comment by dulek** [comment]
Looks good, thanks!
/lgtm

**Comment by eurijon** [comment]
lgtm

**Review by mjpytlak — COMMENTED** [review]
Peer review LGTM @maxwelldb

**Comment by maxwelldb** [comment]
Thanks

**Pull request merged**